mono/tests/Makefile.am

```diff
@@ -65,9 +95,58 @@ STRESS_TESTS_SRC=	\
 	process-stress.cs	\
 	assembly-load-stress.cs
 
+BASE_TEST_MOBILE_STATIC_NOT_SUPPORTED= \
```

Comment:
They're disabled because they attempted to use an appdomain feature or because they emit remoting wrappers that we don't support. I'll go through and confirm that this is true for all of them though.